tests/test_render.py

```diff
@@ -1876,6 +1875,11 @@ def test_hot_reload(self):
                 '#hot-reload-content', 'background-color', 'rgba(255, 0, 0, 1)'
             )
         finally:
+            # load a blank page, so we don't get errant _reload_hash
+            # network failures
+            self.driver.get('about:blank')
```

Comment:
`test_initial_state` is next (alphabetically) after `test_hot_reload`, and its console was filled up with
```
http://localhost:8050/_reload-hash - Failed to load resource: net::ERR_CONNECTION_REFUSED
```
because the hot reloading server had been stopped, and I guess our `clear_log` call isn't happening at the right time in the sequence to wipe these.